frontend/src/pages/Player/MetadataBox/MetadataBox.tsx

```diff
@@ -273,49 +254,6 @@ export const UserDetailsBox = ({
 		return <LoadingBox height={64} />
 	}
 
-	if (!data?.enhanced_user_details) {
-		if (mustUpgradeForClearbit(workspace?.workspace?.plan_tier)) {
-			return (
-				<PaywallTooltip tier={PlanType.Startup}>
-					<div className={style.userEnhancedGrid}>
```

Comment:
Resolves https://linear.app/highlight/issue/HIG-3623/get-rid-of-blurred-user-thingy